safe_transaction_service/contracts/migrations/0007_contract_trusted_for_delegate_call.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.2.9 on 2021-12-02 11:28
+
+from django.db import migrations, models
+
+
+def add_trusted_contracts(apps, schema_editor):
+    Contract = apps.get_model("contracts", "Contract")
+    Contract.objects.filter(
+        address__in=[
+            # Multicalls v1.3.0
+            "0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761",
+            "0x998739BFdAAdde7C933B942a68053933098f9EDa",
+            "0x40A2aCCbd92BCA938b02010E17A5b8929b49130D",
+            "0xA1dabEF33b3B82c7814B6D82A79e50F4AC44102B",
+            # Multicall v1.1.1
+            "0x8D29bE29923b68abfDD21e541b9374737B49cdAD",
```

Comment:
MultiSend v1.1.1 is not used anymore, the interface uses only MultiSendCallOnly 1.3.0. Should it be the only one whitelisted?

Reply:
Regular MultiSend allows DELEGATE_CALL in the underlying transactions and we wanted to prevent it afaik

Reply:
> Regular MultiSend allows DELEGATE_CALL in the underlying transactions and we wanted to prevent it afaik
Not a really big problem, as it cannot execute stuff on the Safe storage

Reply:
> Because if not whitelisted it will not be shown on the history for old transactions

But does the past really matter? 😄 

> Not a really big problem, as it cannot execute stuff on the Safe storage

I think it can.. That's why we switched to CallOnly one.

Reply:
The multisend itself cannot, but it can nest more delegatecalls ... Not sure how we show this in the interface 🤔